[2021-06-25 09:45:14,212] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-01-14T00:00:00+00:00 [queued]>
[2021-06-25 09:45:14,559] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-01-14T00:00:00+00:00 [queued]>
[2021-06-25 09:45:14,560] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:45:14,560] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2021-06-25 09:45:14,560] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:45:14,714] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2020-01-14T00:00:00+00:00
[2021-06-25 09:45:14,794] {standard_task_runner.py:52} INFO - Started process 18920 to run task
[2021-06-25 09:45:14,864] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2020-01-14T00:00:00+00:00', '--job-id', '395', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpb8a5r8q6', '--error-file', '/tmp/tmpx5myv9gi']
[2021-06-25 09:45:14,896] {standard_task_runner.py:77} INFO - Job 395: Subtask task_1
[2021-06-25 09:45:17,100] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2020-01-14T00:00:00+00:00 [running]> on host airflowvm
[2021-06-25 09:45:20,237] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2020-01-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-14T00:00:00+00:00
[2021-06-25 09:45:20,248] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-25 09:45:20,250] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-06-25 09:45:20,445] {subprocess.py:75} INFO - Output:
[2021-06-25 09:45:23,481] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-25 09:45:23,799] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20200114T000000, start_date=20210625T094514, end_date=20210625T094523
[2021-06-25 09:45:24,144] {taskinstance.py:1245} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-06-25 09:45:24,217] {local_task_job.py:151} INFO - Task exited with return code 0
